examples/chatbot/app/views/parts/message.js

```diff
@@ -2,25 +2,25 @@ import React from 'react'
 
 class Message extends React.PureComponent {
   static defaultProps = {
-    user    : '',
-    message : '',
-    pending : false,
-    time    : new Date()
+    user:    '',
+    message: '',
```

Comment:
I feel like aligned objects holds up about as well as `import` blocks. If this is coming from ESLint, and we're voting, can I vote to forego object key alignment altogether?

```js
{
  user: '',
  message: '',
  pending: false,
  time: new Date()
}
```